python/kfserving/kfserving/kfserver.py

```diff
@@ -35,7 +35,7 @@
                     help='The number of works to fork')
 args, _ = parser.parse_known_args()
 
-logging.basicConfig(level=constants.KFSERVING_LOGLEVEL)
+logging.basicConfig(level=constants.KFSERVING_LOGLEVEL, datefmt='%a, %d %b %Y %H:%M:%S', format='%(asctime)s [module:%(module)s][file:%(filename)s][line:%(lineno)d] %(levelname)s %(message)s')
```

Comment:
lint is complaining the line is too long..

Reply:
```
logging.basicConfig(
  level=constants.KFSERVING_LOGLEVEL, 
  datefmt='%a, %d %b %Y %H:%M:%S', 
  format='%(asctime)s [module:%(module)s][file:%(filename)s][line:%(lineno)d] %(levelname)s %(message)s'
)
```
How can we standardize this so that this logging config is inherited? 
Check out this with the root logger: https://stackoverflow.com/questions/47422689/python-logging-how-to-inherit-root-logger-level-handler

Reply:
In fact this is to set the root logger format and will be inherited by all the submodules as long as it uses root logger, (can refer to my output, the tornado and storage format all changed), this is tornado's way of enable_pretty_logging.
Searched code we always used logging.xxx to write the log which in fact uses the root logger. If submodule uses their own logger by something like
```
logger = logging.getlogger("some_name")
logger.xxx
```
This will use their own logger but unless they set format for their own logger, the root logger format will be inherited like the link you posted.

--Thanks
Gang